test/webgl-utils/custom-parameter-settings.js

```diff
@@ -140,66 +140,61 @@ export const GL_PARAMETER_SETTINGS_TWO = {
   [GL.UNPACK_SKIP_IMAGES]: 32
 };
 
-// Note: These settings are same as GL_PARAMETER_SETTINGS_TWO
-// When changing change objects.
-export const GL_PARAMETER_SETTINGS_TWO_ENUM_FUNCTION = {
```

Comment:
Since these test cases are related to the `webgl` layer, I think they should be in `tests/webgl` rather than `tests/webgl-utils`.

Essentially split this into two files.